src/lib/sql/sql_translator.cpp

```diff
@@ -304,7 +303,9 @@ std::shared_ptr<AbstractExpression> SQLTranslator::translate_hsql_expr(const hsq
 
 std::shared_ptr<AbstractLQPNode> SQLTranslator::_translate_insert(const hsql::InsertStatement& insert) {
   const auto table_name = std::string{insert.tableName};
+  AssertInput(!table_name.starts_with(MetaTableManager::META_PREFIX), "Cannot update meta tables");
```

Comment:
I am fine with having the same error message for update, insert, and delete. Just double checking if it was intended or just copy-paste-forget?

Reply:
I took "update" as a synonym for "modify". Will change to the latter.